### ISSUE TITLE
Marshmallow OpenApiConverter does not support str openapi_version 

### ISSUE THREAD

**Issue by paradoxxxzero** [marshmallow-code/apispec#810]
The [mashmallow ext documentation](https://apispec.readthedocs.io/en/latest/api_ext.html#apispec.ext.marshmallow.openapi.OpenAPIConverter) says 
```
openapi_version (Version) – The OpenAPI version to use. Should be in the form ‘2.x’ or ‘3.x.x’ to comply with the OpenAPI standard.
```
but in [APISpec](https://apispec.readthedocs.io/en/latest/api_core.html#module-apispec.core) str is supported which leads to confusion:
```
openapi_version (str) – OpenAPI Specification version. Should be in the form ‘2.x’ or ‘3.x.x’ to comply with the OpenAPI standard.
```

This commit has removed the conversion from str to version instance in marshmallow ext:
https://github.com/marshmallow-code/apispec/commit/d9e7ef9e1225a4d9f1169caba6c7c1235afe8ecd#diff-d2965e63925ff25611aef4a29719e18095c4a035c9a855e65d4d8ea7908298b8L54-L60

which when using str version leads to errors like `AttributeError: 'str' object has no attribute 'major'` here https://github.com/marshmallow-code/apispec/blob/dev/src/apispec/ext/marshmallow/openapi.py#L191 in version 6.0.1

Is it the desired behavior?
Thanks

**Comment by lafrech** [comment]
Right. Sorry about that. Would you like to send a fix (cast to `Version`)?

**Comment by paradoxxxzero** [comment]
Why not :)

**Comment by paradoxxxzero** [comment]
Here you go #811

**Comment by lafrech** [comment]
Fix released in 6.0.2!
